docs/source/format/Columnar.rst

```diff
@@ -618,8 +726,8 @@ for the null struct but they are "hidden" by the struct array's validity
 bitmap. However, when treated independently, corresponding entries of the
 children array will be non-null.
 
-Union Layout
-------------
+Union Layouts
+-------------
```

Comment:
If you want to fix one more issue in this document, there seems to be an issue with the `Variadic Buffers` header:

https://github.com/apache/arrow/actions/runs/6381270929/job/17317395755?pr=37877#step:6:5783

Reply:
Fixed.